src/modules/clock.cpp

```diff
@@ -74,6 +75,13 @@ waybar::modules::Clock::Clock(const std::string& id, const Json::Value& config)
     locale_ = std::locale("");
   }
 
+  if (config_["format-calendar-weeks"].isString()) {
+    weeks_format_ = std::regex_replace(config_["format-calendar-weeks"].asString(), std::regex("\\{\\}"), (first_day_of_week() == date::Monday) ? "{:%V}" : "{:%U}");
+    weeks_format_left_gaps = std::regex_replace(weeks_format_, std::regex(".*<b>|</b>.*|\\{.?+\\}"), "").length();
```

Comment:
I suggest changing regex because format string may not contain `<b>`, or contain other tags inside it, like this: `<b><span color='#99ffdd'>{:%U}</span></b>`.
For example `"</?[^>]+>|\\{.?+\\}"` can be used.

Reply:
Thank you @prohornikitin , I'll check tonight,

Reply:
@prohornikitin  good point. Thanks. Took into the PR